### PR TITLE
fix issue #158

### PR DESCRIPTION
**Pull request by sandheep45**

### CODE REVIEW

**Comment by sandheep45** [comment]
@prasenjeet-symon i was getting the error because my migration file was not updated as i was not aware of this. I kindly request to give a throught again about ignoring migration files. Thank you

**Comment by prasenjeet-symon** [comment]
> @prasenjeet-symon i was getting the error because my migration file was not updated as i was not aware of this. I kindly request you to give a throught again about ignoring migration files. Thank you

Do not ignore the migration file. If in future someone will complain about the migration issue we will think about that later on.

**Comment by sandheep45** [comment]
> > @prasenjeet-symon i was getting the error because my migration file was not updated as i was not aware of this. I kindly request you to give a throught again about ignoring migration files. Thank you
> 
> Do not ignore the migration file. If in future someone will complain about the migration issue we will think about that later on.

include or ignore?

File: packages/server/prisma/seed.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { createMultipleUsers } from "../src/seeder/utils";
+
+const main = async () => {
+  const newUsers = await createMultipleUsers(14)
+  return newUsers
+};
+
+main().then(console.log).catch(console.log);
```
Comment (prasenjeet-symon) [inline packages/server/prisma/seed.ts:8]: We need to rethink the approach of invoking the main function directly. The original index.ts file, structured as a class, had the advantage of not running on server startup. Running migrations upon server startup is not our preferred method. Instead, we should trigger migrations once a user is created by our contributors. This approach allows us to focus on a specific user (their email ID) and ensures that all activities revolve around this user as the central point. This user-centric strategy aligns better with our goals.

File: packages/server/package.json [hunk 2]
```diff
@@ -9,6 +9,9 @@
     "migrate": "npx prisma migrate dev",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
+  "prisma": {
+    "seed": "ts-node prisma/seed.ts"
```
Comment (prasenjeet-symon) [inline packages/server/package.json:13]: We need a user of focus

Comment (sandheep45) [reply packages/server/package.json:13]: i didn't get that

File: packages/server/package.json [hunk 3]
```diff
@@ -17,11 +20,13 @@
     "axios": "^1.5.0",
     "cors": "^2.8.5",
     "dotenv": "^16.3.1",
+    "enforce-unique": "^1.2.0",
```
Comment (prasenjeet-symon) [inline packages/server/package.json:23]: Just for a single purpose do not install new lib . Come up with your typescript solution.

Comment (prasenjeet-symon) [reply packages/server/package.json:23]: Let it be. It's alright

**Review by prasenjeet-symon — APPROVED** [review]
Looks good but needs a little bit of changes to the seeder file located in the Prisma folder.

**Comment by prasenjeet-symon** [comment]
> > > @prasenjeet-symon i was getting the error because my migration file was not updated as i was not aware of this. I kindly request you to give a throught again about ignoring migration files. Thank you
> > 
> > 
> > Do not ignore the migration file. If in future someone will complain about the migration issue we will think about that later on.
> 
> include or ignore?

Include it into source code

**Review by prasenjeet-symon — APPROVED** [review]

**Pull request merged**